spyder/plugins/console/widgets/console.py

```diff
@@ -147,7 +147,11 @@ class ConsoleBaseWidget(TextEditBaseWidget):
     """Console base widget"""
     BRACE_MATCHING_SCOPE = ('sol', 'eol')
     COLOR_PATTERN = re.compile(r'\x01?\x1b\[(.*?)m\x02?')
-    exception_occurred = Signal(str, bool)
+
+    # Signals
+    sig_exception_occurred = Signal(str, bool)
```

Comment:
Please add a comment above to describe what this signal does.

Reply:
Fixed